partials/footer.php

```diff
@@ -6,6 +6,7 @@
 ?>
 
 <footer class="footer">
+<div class="footernotcopyright">
```

Comment:
I added the extra container so that I could put copyright on its own line below. I have formatted the footer to go in a row but this includes the copyright as well. I tried the width 100% but it still remains in the same line.

Reply:
Try settings `flex-wrap: wrap`, that should allow the flexbox items to flow down onto multiple rows if they don't fit. You may even be able to set it from the start and it'll bring things onto multiple lines when they fit.

Have a look at the footer on this website - https://thirdavenuesurgery.com.au/, the styles are here: https://github.com/aimhigherweb/keywain-wordpress-template/blob/prod/src/scss/partials/footer.scss